src/PowerShell/Help/Microsoft.WinGet.Client/Add-WinGetSource.md

```diff
@@ -0,0 +1,100 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+online version:
+schema: 2.0.0
+---
+
+# Add-WinGetSource
+
+## SYNOPSIS
+Add a new source.
+
+## SYNTAX
+
+```
+Add-WinGetSource -Name <String> -Argument <String> [-Type <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Add a new WinGet source.
+
+## EXAMPLES
+
+### Example 1: Add new REST source named mysource
+
+```powershell
+PS C:\> Add-WinGetSource -Name mysource -Argument https://contoso.com/ -Type Microsoft.Rest
+```
+
+This example adds a new REST based source to WinGet named "mysource" with the root URL https://contoso.com/.
+
+> **This example is illustrative and will not succeed because WinGet will not add a source that does not properly respond to the WinGet REST API**
```

Comment:
```suggestion
> **This example is illustrative and will not succeed because WinGet will not add a source that does not properly respond to the WinGet REST API.**
```

nit, but while we're at it I'd add the period just like below, in line 40.

Reply:
https://github.com/microsoft/winget-cli/pull/4674/commits/1f92ce862400271def035cadace5c40eabc8de21 resolved this 👌